dealership/vehicles.py

```diff
@@ -1,15 +1,38 @@
 class Vehicle(object):
```

Comment:
Similar to the `Contract` class, there is no `NotImplemented` exception when the `Vehicle` class is instantiated. Once again, this is completely up to you. If you would like to make the class more abstract, then you use utilize `NotImplemented` errors here.